### ISSUE TITLE
Preview stops after zooming

### ISSUE THREAD

**Issue by BlueBazze** [ThibaultBee/StreamPack#45]
TLDR: A single device have experienced a malfunction while using the zooming functionality. If this should be a bigger issue it should be looked at further.

If you come across this issue and you have a Nokia device, it would be nice if you can test this issue so we can get some data on how big the issue is.

## Issue description
When zooming, in this case using the StreamPack demo-camera app - When setting the zoomRatio, the preview appears to stop displaying the camera view.
With the error message `E/CameraDeviceCallback: Camera 0 is in error 4`

At this point we dont know the scope of this issue.
We want to figure out the cause of this issue.
These are the candidates that we can come up with.
- Android 11
- Nokia's android distro
- The model Nokia T20
- StreamPack
- The spcific device that i have ahold of

## Testing
First you confirm that the issue you are experiencing is the same as described here.
If this is the same issue. Then please provide the following information.

- Device name (model)
- Android version
- The zoomRatioRange, this can be printed by inserting the following line in the set part of zoomRatio
`if(value > 1f && zoomRatioRange.value != null) Log.d("STREAMPACK - ZoomRange:", zoomRatioRange.value!!.lower.toString() + " - " + zoomRatioRange.value!!.upper.toString())` - Which is also referenced in the link just below
- Put a breakpoint at the set part of zoomRatio in the demo app. And follow the stack until the preview stops and note where the preview stops https://github.com/ThibaultBee/StreamPack/blob/main/demos/camera/src/main/java/io/github/thibaultbee/streampack/app/ui/main/PreviewViewModel.kt#L200
- A thread dump from the debugging
- The error you get after the preview has stopped

### Nokia T20
- Device Name: Nokia T20
- Android version: Android 11
- ZoomRatioRange: 1.0 - 4.0 (lower - upper)
- Breaks when returning on line: https://github.com/ThibaultBee/StreamPack/blob/fa05dcc939ed317919c2a297e9d95dea6b5cf2f0/core/src/main/java/io/github/thibaultbee/streampack/internal/sources/camera/CameraExecutorManager.kt#L71
- Thread dump: https://pastebin.com/DP3sxUTK
- Error `E/CameraDeviceCallback: Camera 0 is in error 4`

### Nokia 7.1 - Android 10
Works without any similarities to the issue on Nokia T20, and works without any errors.

## Comment
This issue has been discussed here: 
- https://github.com/apivideo/api.video-reactnative-live-stream/pull/32#issuecomment-1219550338
- https://github.com/apivideo/api.video-android-live-stream/pull/9#issuecomment-1217992368
Any further discussion should continue here.

As of now this issue has only been experienced on a single device, and not reproduced by any other devices. Which would not be considered a critical fault with StreamPack. But for instance if the scope of this issue was affecting all Nokia devices on Android 11 and above, it might have to get looked at.

**Comment by ThibaultBee** [comment]
Hi there,
Stalled.
Please reopen it if you got more info.
